ql/instruments/creditdefaultswap.hpp

```diff
@@ -184,7 +217,8 @@ namespace QuantLib {
                                const Handle<YieldTermStructure>& discountCurve,
                                const DayCounter& dayCounter,
                                Real recoveryRate = 0.4,
-                               Real accuracy = 1.0e-6) const;
+                               Real accuracy = 1.0e-6,
+                               bool useIsdaEngine = false) const;
```

Comment:
Maybe these bool could be changed to an enumeration? It would allow for future extension.

Reply:
I've added an enum CdsPricingEngine inside the CreditDefaultSwap. I'm not sure this was the best place to put it.